Libraries/Components/Button.windows.js

```diff
@@ -116,6 +147,32 @@ class Button extends React.Component {
     );
     const formattedTitle = Platform.OS === 'android' ? title.toUpperCase() : title;
     const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    let content =
+        <View style={buttonStyles}>
+          <Text style={textStyles} disabled={disabled}>{formattedTitle}</Text>
+        </View>;
+
+    if (Platform.OS === "windows") {
+      const tabIndex = this.props.tabIndex || 0;
+      const windowsTabFocusable = !disabled && tabIndex >= 0;
+      content =
+        <FocusableWindows
+          ref={this._setFocusableRef}
+          disabled={disabled}
+          isTabStop={windowsTabFocusable}
+          tabIndex={tabIndex}
+          disableSystemFocusVisuals={this.props.disableSystemFocusVisuals}
+          handledKeyDownKeys={DOWN_KEYCODES}
+          handledKeyUpKeys={UP_KEYCODES}
+          onKeyDown={this._onKeyDown}
+          onKeyUp={this._onKeyUp}
+          onFocus={this._onFocus}
+          onBlur={this._onBlur}
+        >
+          {content}
+      </FocusableWindows>;
```

Comment:
Any reason to double nest the canvases here? I.e., <FocusableWindows><View>...</View></FocusableWindows>? Couldn't we just apply the `buttonStyles` to the control?

Reply:
Oh, FocusableWindows is not a View. I know it internally has a Canvas as content of that UserControl, but it doesn't have a BorderCanvas functionality/properties/etc.

---
In reply to: [159510139](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r159510139) [](ancestors = 159510139)